src/templates/users/user_detail.html

```diff
@@ -3,25 +3,48 @@
 {% load static %}
 
 {% block title %}
-  User:
-  {{ object.name }}
+  My Details:
 {% endblock title %}
 {% block content %}
   <div class="container">
     <div class="row">
       <div class="col-sm-12">
-        <h2>{{ object.name }}</h2>
+        <h2>My Details:</h2>
       </div>
     </div>
     {% if object == request.user %}
       <!-- Action buttons -->
+      <style>
+        .list-item {
+          margin-bottom: 20px;
+          list-style-type: none;
+        }
+      </style>
       <div class="row">
         <div class="col-sm-12">
-          <a class="btn btn-primary" href="{% url 'users:update' %}" role="button">My Info</a>
-          <a class="btn btn-primary"
-             href="{% url 'account_email' %}"
-             role="button">E-Mail</a>
-          <!-- Your Stuff: Custom user template urls -->
+          <ul>
+            <li class="list-item">
+              User Display Name: {{ object.name }}
+              <a class="btn btn-primary" href="{% url 'users:update' %}" role="button">Update Name</a>
+            </li>
+            <li class="list-item">
+              Email: {{ object.email }}
+              <a class="btn btn-primary"
+                 href="{% url 'account_email' %}"
+                 role="button">Update E-Mail</a>
+            </li>
+            <li class="list-item">
+              Password:
+              <a class="btn btn-primary"
+                 href="{% url 'users:password_reset' %}"
+                 role="button">Reset Password</a>
+            </li>
+            <li class="list-item">Bio: {{ object.userprofile.bio }}</li>
+            <li class="list-item">Date Joined: {{ object.date_joined }}</li>
+            <li class="list-item">Is Admin: {{ object.is_staff }}</li>
+            <li class="list-item">Friends: {{ object.userprofile.friends }}</li>
```

Comment:
If this field is none, it should just be blank. I'm currently getting "users.User.None".